dev-tools/omdb/src/bin/omdb/db.rs

```diff
@@ -3672,6 +3700,140 @@ async fn cmd_db_region_find_deleted(
     Ok(())
 }
 
+async fn cmd_db_dry_run_region_allocation(
+    opctx: &OpContext,
+    datastore: &DataStore,
+    args: &DryRunRegionAllocationArgs,
+) -> Result<(), anyhow::Error> {
+    let volume_id = VolumeUuid::new_v4();
```

Comment:
Should this be controllable by a caller, and defaults to `new_v4()`?

Reply:
Actually yeah, especially to test the "please allocate N more than currently exist" case, good point! Done in 427ef4442